lib/OpenQA/Controller/API/V1/Job.pm

```diff
@@ -206,7 +206,7 @@ sub restart {
     }
 
     my @res = OpenQA::Scheduler::job_restart($target);
-    $self->render(json => {result => \@res});
+    $self->render(json => {result => $res[0], test_url => $self->url_for('test', testid => $res[0]) });
```

Comment:
I don't know if we use array version anywhere, but to be on safe side I would keep arrays. Or if you don't want to, then remove the every_param part.